src/verinfast/agent.py

```diff
@@ -667,6 +676,11 @@ def main():
     agent = Agent()
     try:
         agent.scan()
+        with open(f"{agent.config.output_dir}/results.html", "w") as f:
+            jinja_env = Environment(loader=FileSystemLoader("templates/"))
+            jinja_env.globals.update(zip=zip)
+            output = jinja_env.get_template("results.j2").render(template_definition)
+            f.write(output)
```

Comment:
Getting this error when I run this:
`Traceback (most recent call last):
  File "/home/stconrad/.local/bin/verinfast", line 8, in <module>
    sys.exit(main())
  File "/home/stconrad/chris/verinfast/src/verinfast/agent.py", line 688, in main
    raise e
  File "/home/stconrad/chris/verinfast/src/verinfast/agent.py", line 682, in main
    output = jinja_env.get_template("results.j2").render(template_definition)
  File "/home/stconrad/.local/lib/python3.10/site-packages/jinja2/environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "/home/stconrad/.local/lib/python3.10/site-packages/jinja2/environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/home/stconrad/.local/lib/python3.10/site-packages/jinja2/loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/stconrad/.local/lib/python3.10/site-packages/jinja2/loaders.py", line 218, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: results.j2`

Reply:
What is your output for `pwd`?

Reply:
change to abspath resolves this and covered by new unit test.